fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java

```diff
@@ -56,6 +57,10 @@
         return idTranslator;
     }
 
+    protected Node asNode(final FedoraResource resource) {
```

Comment:
Obviously, this is a good shortcut. Maybe a quick JavaDoc here just for those unfamiliar with the codebase?